deploy/template.yaml

```diff
@@ -209,39 +209,37 @@ Resources:
               SCENARIO: all
               K6_DYNATRACE_DASHBOARD_ID: 1e845440-d013-4472-9d65-2ea21a5cb41a
             parameter-store:
+              ACCOUNT_APP_KEY: "/perfTest/account/accounts/appKey"
               ACCOUNT_APP_PASSWORD: "/perfTest/account/testUserPassword"
               ACCOUNT_APP_PASSWORD_NEW: "/perfTest/account/testUserNewPassword"
-              ACCOUNT_APP_KEY: "/perfTest/account/accounts/appKey"
               ACCOUNT_BASE_URL: "/perfTest/account/authentication/baseUrl"
               ACCOUNT_CURR_PHONE: "/perfTest/account/accounts/currPhone"
               ACCOUNT_EMAIL_OTP: "/perfTest/account/fixedEmailOTP"
               ACCOUNT_HOME_URL: "/perfTest/account/accounts/homeUrl"
               ACCOUNT_NEW_PHONE: "/perfTest/account/accounts/newPhone"
               ACCOUNT_PHONE_OTP: "/perfTest/account/fixedPhoneOTP"
-              ACCOUNT_SIGNIN_URL: "/perfTest/account/accounts/signInUrl"
               ACCOUNT_RP_STUB: "/perfTest/account/authentication/rpStub"
-              IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
+              ACCOUNT_SIGNIN_URL: "/perfTest/account/accounts/signInUrl"
               IDENTITY_ADDRESS_ENV_NAME: "/perfTest/identity/orange/addressEnvName"
+              IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
               IDENTITY_CORE_STUB_URL: "/perfTest/identity/coreStubUrl"
               IDENTITY_CORE_URL: "/perfTest/identity/coreUrl"
               IDENTITY_DRIVING_ENDPOINT: "/perfTest/identity/lime/drivingEndpoint"
               IDENTITY_DRIVING_URL: "/perfTest/identity/lime/drivingUrl"
               IDENTITY_FRAUD_URL: "/perfTest/identity/lime/fraudUrl"
-              IDENTITY_KBV_URL: "/perfTest/identity/orange/kbvUrl"
               IDENTITY_KBV_ENV_NAME: "/perfTest/identity/orange/kbvEnvName"
+              IDENTITY_KBV_URL: "/perfTest/identity/orange/kbvUrl"
               IDENTITY_KIWI_URL: "/perfTest/identity/kiwi/url"
               IDENTITY_ORCH_STUB_URL: "/perfTest/identity/orchStubUrl"
               IDENTITY_PASSPORT_URL: "/perfTest/identity/lime/passportUrl"
-              IDENTITY_STUB_USERNAME: "/perfTest/identity/coreStubUsername"
               IDENTITY_STUB_PASSWORD: "/perfTest/identity/coreStubPassword"
+              IDENTITY_STUB_USERNAME: "/perfTest/identity/coreStubUsername"
               K6_DYNATRACE_APITOKEN: "/perfTest/dynatraceApiToken"
               K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
               MOBILE_BACKEND_URL: "/perfTest/mobile/backendUrl"
```

Comment:
If the folders are dcmaw, maybe this should be `/perfTest/dcmaw/backendUrl`

Reply:
I think if we keep this as `/perfTest/mobile/backendUrl` we can keep parameters in the form of
- `/perfTest/<pod_name>/<variable_name>` or
- `/perfTest/<pod_name>/<team_name>/<variable_name>`

and then rename the `dcmaw` folder to `mobile` to align to the name of the pod?